packages/plugin-core/src/settings.ts

```diff
@@ -24,7 +24,10 @@ import { DendronExtension, getDWorkspace } from "./workspace";
 export { Snippets };
 
 export class Extensions extends EngineExtension {
-  static getVSCodeExtnsion() {
+  /**
+   * Get Dendron recommended extensions
+   */
+  static getDendronVSCodeExtnsion() {
```

Comment:
nit, carry over sp on 'Extension'.  Also, if this gets recommended extensions, should we just name the function that way? `getDendronRecommendedExtensions`

Reply:
renamed. this does recommended extensions to install AND remove